rotkehlchen/tests/unit/test_evm_contracts.py

```diff
@@ -33,3 +34,30 @@ def test_evm_abi_data(globaldb):
             assert serialized_abi == entry[1]
             assert entry[1] not in abis_set
             abis_set.add(entry[1])
+
+
+def test_fallback_to_packaged_db(ethereum_inquirer: 'EthereumInquirer'):
+    """
+    Test that if a contract / abi is missing in the globaldb, it is searched in the packaged db.
+    """
+    with GlobalDBHandler().conn.write_ctx() as cursor:
+        # Delete one contract and its abi
+        cursor.execute(
+            'SELECT contract_data.address, contract_abi.value FROM contract_data INNER JOIN '
+            'contract_abi ON contract_data.abi=contract_abi.id WHERE chain_id=1 LIMIT 1',
+        )
+        (address, abi) = cursor.fetchone()  # There has to be at least one entry
+        cursor.execute('DELETE FROM contract_data WHERE address=? AND chain_id=1', (address,))
```

Comment:
Hmm I think you need to work with a specific contract and ABI. This won't work forever since an ABI may be reused by other contracts. So you would botch those contracts. Deleting an ABI sets it to null. Won't raise a FOREIGN key error.

On the other hand this is just for a test ... so probably does not matter here.

Reply:
all right. Fine.